ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -550,18 +590,43 @@ extension ClientQueryViewController : OCQueryDelegate {
 // MARK: - SortBar Delegate
 extension ClientQueryViewController : SortBarDelegate {
 	func sortBar(_ sortBar: SortBar, leftButtonPressed: UIButton) {
-		print("LOG ---> left button pressed")
+		let createFolderVC = NamingViewController(with: core, defaultName: "Unknown Folder".localized, stringValidator: { name in
```

Comment:
Like @jesmrec I think that "New folder" is what a user would usually expect here.

Reply:
Definitely right. 
I just copied here what files app does but _New Folder_ is a lot better. I'll change it.